app/src/main/res/values/strings_about.xml

```diff
@@ -37,15 +37,15 @@
     <string name="about_label_translate">Translate</string>
 
     <string name="about_desc_report_bugs">Report bugs or request new features</string>
-    <string name="about_desc_translate">Help to translate andOTP to your language</string>
+    <string name="about_desc_translate">Help us translate andOTP to your own native language</string>
 
     <string name="about_title_special_features">Special features</string>
     <string name="about_dialog_special_features">Are you sure you want to enable the special
-        features? Most of them are non-standard OTP algorithms which were implemented on special
+        features of andOTP? Most of them are non-standard OTP algorithms which were implemented on special
         request. They are <b>not officially supported</b> and can come with <b>certain
         limitations</b>, please read (and understand) the page \"Special features\" in the Github
-        wiki to learn more before using them.</string>
-    <string name="about_toast_special_features">Special features enabled</string>
-    <string name="about_toast_special_features_enabled">Special features already enabled</string>
-    <string name="about_toast_copied_to_clipboard">Donation link copied to clipboard</string>
+        wiki so as to learn more about these features before using them.</string>
+    <string name="about_toast_special_features">Special features has been enabled</string>
```

Comment:
I think it should be `have been` (plural)

Reply:
> I think it should be `have been` (plural)

A native English speaker here. You are correct.

Reply:
Ok @Maikuolan . Point noted. Would effect the chances ASAP

This particular error was typographical. 
My apologies!